src/cli/view.rs

```diff
@@ -28,7 +28,8 @@ pub fn display_data<W: Write>(
             for b in d {
                 match *b {
                     // null byte
-                    0x00 | 0xff => write!(output, "{}  ", "•".bright_black()).unwrap(),
+                    0x00 => write!(output, "{} ", "··".bright_black()).unwrap(),
```

Comment:
Very nice idea put a minor point for 0 and a full one for 255. This idea is aproved.